deploy/kubeturbo/templates/deployment.yaml

```diff
@@ -40,13 +40,13 @@ spec:
           - --k8sVersion=1.5
           {{- end }}
           volumeMounts:
-          - name: turbo-volume-{{ .Release.Name }}
```

Comment:
I think this would create a conflict where multiple kubeturbo instances are deployed in the same cluster